azurerm/resource_arm_mysql_configuration.go

```diff
@@ -51,9 +52,21 @@ func resourceArmMySQLConfigurationCreate(d *schema.ResourceData, meta interface{
 	name := d.Get("name").(string)
 	resourceGroup := d.Get("resource_group_name").(string)
 	serverName := d.Get("server_name").(string)
-
 	value := d.Get("value").(string)
 
+	if requireResourcesToBeImported && d.IsNewResource() {
```

Comment:
(as in the other PR) the MySQL/PostgreSQL Configuration resources can't be imported since they're effectively virtual resources - so can we remove this?